### DISCUSSION TITLE
Bug-report: labels

### DISCUSSION THREAD

**Discussion by boozook** [codemagic-ci-cd/codemagic-docs#2072] [Q&A] [answered]
### Use case

Get a permanently broken app! 🎉

```
[Error] TypeError: o.labels.sort is not a function. (In 'o.labels.sort((function(e,t){return e.localeCompare(t,void 0,{sensitivity:"base"})}))', 'o.labels.sort' is undefined)
...
```

### Idea

```Text
Send workflow trigger via API with `"labels": "string!"`.
There is no any kind of input validation.
```

**Comment by himesh-cm** [comment]
Hi @boozook,
Thank you for reporting this issue. I have forwarded the information to the developers. In the meantime, could you please provide your team ID so we can fix the problem from our end?

**Reply by himesh-cm** [reply]
I tried the below request.
```
{
  "appId": "64cb6e1b9f2e2aa4e63b619b",
  "workflowId": "ios-native-workflow",
  "branch":"main",
  "labels": "stringTest!"
}
```

For me, the build is triggered, and the app breaks, as you mentioned, but I do not get the `o.labels.sort is not a function` error. Can you help me with the request so we can check that?

**Reply by boozook** [reply]
@himesh-cm, thanks for your answer!
Seems to my `teamId` is `650d7407e790b7bbb49787d3`.

**Reply by boozook** [reply]
@himesh-cm, here is request example:
```bash
echo '{ "appId": "<APPID>", "workflowId": "<ID>", "labels": "JUST_THIS", "branch": "main" }' | \
curl -s \
     -H "Content-Type: application/json" \
     -H "x-auth-token: <API_TOKEN>" \
     -X POST https://api.codemagic.io/builds \
     -d @- | \
     jq -Rr 'fromjson | .buildId'
```

**Reply by himesh-cm** [reply, marked as answer]
Hi @boozook,
Your `/builds` page should work now. The team is working on fixes.

**Reply by himesh-cm** [reply]
The issue has been fixed, too.
While I’m closing our discussion for now, if you have any other questions or suggestions, feel free to reply back or start a new discussion anytime. Have a nice day!
